app/models/table.rb

```diff
@@ -26,6 +26,7 @@ class Table < ApplicationRecord
   has_one :result, through: :result_table
   has_many :report_elements, inverse_of: :table, dependent: :destroy
 
+  before_save :convert_metadata_plate_template
```

Comment:
why do you need this conversion?

Reply:
We need this conversion so we handle correctly "true" and true values for plateTemplates in case someone creates like this, so we can handle rendering from here because we expect to have this field boolean value.

Reply:
why not handle this specific case in the controller?